backend/src/api/transactions/transaction.model.js

```diff
@@ -28,13 +28,49 @@ const transactionSchema = new Schema(
 );
 
 transactionSchema.statics.createTransaction = createTransaction;
+transactionSchema.statics.getCurrentMonthBalance = getCurrentMonthBalance;
 
 const TransactionModel = mongoose.model('Transaction', transactionSchema);
 
 async function createTransaction(info) {
   return this.create(info);
 }
 
+async function getCurrentMonthBalance(userId) {
+  const date = new Date();
+  const startMonth = new Date(date.getFullYear(), date.getMonth(), 1);
```

Comment:
it won't work correctly in all cases. Better to take month order number (11 for November, for instance) and do something like that lower in match - `$eq: [{ $month: '$transactionDate' }, monthNumber]`